media-gfx/apng2gif/apng2gif-1.7.recipe

```diff
@@ -0,0 +1,54 @@
+SUMMARY="Converts APNG animations into GIF format"
+DESCRIPTION="This program converts APNG animations into animated GIF format. \
+It performs the reverse of gif2apng."
+HOMEPAGE="http://apng2gif.sourceforge.net/"
+COPYRIGHT="2009-2015 Max Stepin"
+LICENSE="Zlib"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/apng2gif/apng2gif-$portVersion-src.zip"
+CHECKSUM_SHA256="6c7b2cce851378f2fc8f6b490ce098e9cf0038aa05da7099771c8be94620e6ca"
+SOURCE_DIR=""
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+if [ "$targetArchitecture" != x86_gcc2 ]; then
+	commandSuffix=$secondaryArchSuffix
+	commandBinDir=$binDir
+else
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
```

Comment:
Hmm, let's drop this block (in both apng2gif and gif2apng).
There are several reasons for this:
* We are not using commandSuffix.
* We are only using commandBinDir in one place, INSTALL, and we can replace **`commandBinDir`** by **`$prefix/bin`** for this recipe on all architectures, both primary and secondary.
* This recipe does not provide any lib, so after we drop x86_gcc2 (some time in the future) we will be able to completely drop support for secondary architectures for this recipe.